src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsaccreditedprogrammesapi/controller/ProgrammesController.kt

```diff
@@ -2,47 +2,47 @@ package uk.gov.justice.digital.hmpps.hmppsaccreditedprogrammesapi.controller
 
 import org.springframework.http.ResponseEntity
 import org.springframework.stereotype.Service
-import uk.gov.justice.digital.hmpps.hmppsaccreditedprogrammesapi.api.ProgrammesApiDelegate
-import uk.gov.justice.digital.hmpps.hmppsaccreditedprogrammesapi.api.model.Programme
-import uk.gov.justice.digital.hmpps.hmppsaccreditedprogrammesapi.api.model.ProgrammePrerequisite
-import java.util.*
+import uk.gov.justice.digital.hmpps.hmppsaccreditedprogrammesapi.api.CoursesApiDelegate
+import uk.gov.justice.digital.hmpps.hmppsaccreditedprogrammesapi.api.model.Course
+import uk.gov.justice.digital.hmpps.hmppsaccreditedprogrammesapi.api.model.CoursePrerequisite
+import java.util.UUID
 
 @Service
-class ProgrammesController : ProgrammesApiDelegate {
-  override fun programmesGet(): ResponseEntity<List<Programme>> {
+class ProgrammesController : CoursesApiDelegate {
+  override fun coursesGet(): ResponseEntity<List<Course>> {
     return ResponseEntity.ok(
       listOf(
-        Programme(
+        Course(
           id = UUID.randomUUID(),
           name = "Thinking Skills Programme",
           programmeType = "TypeA",
           description = "Thinking Skills Programme (TSP) is for adult men and women with a medium or high risk of re offending...",
           programmePrerequisites = listOf(
-            ProgrammePrerequisite(key = "gender", value = "female"),
-            ProgrammePrerequisite(key = "risk score", value = "ORGS: 50+"),
-            ProgrammePrerequisite(key = "offence type", value = "some offence here"),
+            CoursePrerequisite(key = "gender", value = "female"),
+            CoursePrerequisite(key = "risk score", value = "ORGS: 50+"),
+            CoursePrerequisite(key = "offence type", value = "some offence here"),
           ),
         ),
-        Programme(
+        Course(
           id = UUID.randomUUID(),
           name = "Becoming new me +",
           programmeType = "TypeB",
           description = "Becoming new me + is for adult men and women with a medium or high risk of re offending...",
           programmePrerequisites = listOf(
-            ProgrammePrerequisite(key = "gender", value = "female"),
-            ProgrammePrerequisite(key = "risk score", value = "ORGS: 50+"),
-            ProgrammePrerequisite(key = "offence type", value = "some offence here"),
+            CoursePrerequisite(key = "gender", value = "female"),
```

Comment:
Sorry, this wasn't in the ticket, but we've renamed `key` and `value` to `name` and `description` - is that something we could do here too?